build.gradle

```diff
@@ -82,7 +82,6 @@ dependencies {
     testImplementation 'junit:junit:4.12'
 
     errorprone group: 'com.google.errorprone', name: 'error_prone_core', version: '2.10.0'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
I removed this because the kotlin plugin already automatically imports a relevant version